packages/grafana-ui/src/components/Tabs/Tab.tsx

```diff
@@ -73,12 +72,24 @@ const getTabStyles = stylesFactory((theme: GrafanaTheme2) => {
       padding: ${theme.spacing(1.5, 2, 1)};
       display: block;
       height: 100%;
+
       svg {
         margin-right: ${theme.spacing(1)};
       }
 
       &:focus-visible {
-+        ${getFocusStyles(theme)}
+        ${getFocusStyles(theme)}
+      }
+
+      &::before {
```

Comment:
The idea behind this change is to remove code duplication and only change the background depending on the tab state.